routes/project/kubernetes_cluster.rb

```diff
@@ -2,13 +2,13 @@
 
 class Clover
   hash_branch(:project_prefix, "kubernetes-cluster") do |r|
+    r.get true do
+      kubernetes_cluster_list
+    end
+
     r.web do
       r.is do
-        r.get do
-          kubernetes_cluster_list
-        end
-
-        r.post do
+        r.post true do
```

Comment:
`r.is` is for when you have different HTTP verbs handling the same route, and want to do the check that you've routed the entire path only once.  In this case, you only have 1 route, so it isn't needed.  You can remove the `r.is`, and just have `r.post true`.  Note that if you are using `r.get` or `r.post` inside `r.is`, you do not need to provide the `true` argument.